docs/tutorial/collation.txt

```diff
@@ -66,7 +66,7 @@ Collation Parameters
    ]
 
 The only required parameter is ``locale``, which the server parses as an `ICU
-format locale ID <http://userguide.icu-project.org/locale>`_. For example, set
+format locale ID <https://userguide.icu-project.org/locale>`_. For example, set
```

Comment:
```suggestion
format locale ID <https://unicode-org.github.io/icu/userguide/locale/>`_. For example, set
```

The original link, http://userguide.icu-project.org/locale, redirects to https://sites.google.com/site/icuprojectuserguide/locale, which in turns points to the above URL.